infra/modules/service/events_jobs.tf

```diff
@@ -40,29 +40,18 @@ resource "aws_cloudwatch_event_target" "document_upload_jobs" {
 
```

Comment:
risk of messing up the scheduled jobs

Reply:
Tested all the scheduled jobs, they're working fine.

[Except this one](https://betagrantsgov.slack.com/archives/C05TSL64VUH/p1741624181477479), which isn't broken because of infra changes.